src/client/graphics/layers/PlayerInfoOverlay.ts

```
@@ -217,10 +219,19 @@
             : "text-white"}"
         >
           ${player.flag()
-            ? html`<img
-                class="h-8 mr-1 aspect-[3/4]"
-                src=${"/flags/" + player.flag() + ".svg"}
-              />`
+            ? player.flag().startsWith("ctmfg")
+              ? html`<div
+                  class="h-8 mr-1 aspect-[3/4] player-flag"
+                  ${ref((el) => {
+                    if (el != null) {
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)